zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -178,29 +134,63 @@ Open Issues
 > TODO: Remove this section once a decision is made.
```

Comment:
This section will need to become a Rationale section briefly saying why we chose these parameters and this formula.